src/eterna/AnnotationManager.ts

```diff
@@ -662,14 +648,12 @@ export default class AnnotationManager {
      * currently modifiable category
      */
     public get activeLayers() {
```

Comment:
This whole function can be removed now right?

Reply:
Indeed